test/testfiles/igem_ludox_test.nt

```diff
@@ -334,23 +334,18 @@
 <https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/ObjectFlow7> <http://sbols.org/v3#displayId> "ObjectFlow7" .
 <https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/ObjectFlow7> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://bioprotocols.org/uml#ObjectFlow> .
 <https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/ObjectFlow7> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://sbols.org/v3#Identified> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger1> <http://bioprotocols.org/uml#integerValue> "1"^^<http://www.w3.org/2001/XMLSchema#integer> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger1> <http://sbols.org/v3#displayId> "LiteralInteger1" .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger1> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://bioprotocols.org/uml#LiteralInteger> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger1> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://sbols.org/v3#Identified> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger2> <http://bioprotocols.org/uml#integerValue> "1"^^<http://www.w3.org/2001/XMLSchema#integer> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger2> <http://sbols.org/v3#displayId> "LiteralInteger2" .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger2> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://bioprotocols.org/uml#LiteralInteger> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger2> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://sbols.org/v3#Identified> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1> <http://bioprotocols.org/uml#direction> <http://bioprotocols.org/uml#out> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1> <http://bioprotocols.org/uml#isOrdered> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1> <http://bioprotocols.org/uml#isUnique> "true"^^<http://www.w3.org/2001/XMLSchema#boolean> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1> <http://bioprotocols.org/uml#lowerValue> <https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger2> .
-<https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1> <http://bioprotocols.org/uml#upperValue> <https://bbn.com/scratch/iGEM_LUDOX_OD_calibration_2018/Parameter1/LiteralInteger1> .
```

Comment:
We're losing the upper and lower bound information on parameters here; see comment below for what I believe the reason is.